packages/next/client/image.tsx

```diff
@@ -341,6 +342,34 @@ function cloudinaryLoader({ root, src, width, quality }: LoaderProps): string {
 }
 
 function defaultLoader({ root, src, width, quality }: LoaderProps): string {
+  if (process.env.NODE_ENV !== 'production') {
+    const missingValues = []
+
+    // these should always be provided but make sure they are
+    if (!src) missingValues.push('src')
+    if (!width) missingValues.push('width')
+
+    if (missingValues.length > 0) {
+      throw new Error(
+        `Next Image optimizer requires ${missingValues.join(
```

Comment:
```suggestion
        `Next.js Image Optimization requires ${missingValues.join(
```

The docs in #18107 never use "Optimizer", we should be consistent and use "Optimization"